p2p/agent.go

```diff
@@ -109,6 +108,10 @@ type (
 		Self() ([]multiaddr.Multiaddr, error)
 		// Neighbors returns the neighbors' peer info
 		Neighbors(ctx context.Context) ([]peer.AddrInfo, error)
+		// ConnectedPeers returns the connected peers' info
```

Comment:
we can remove `Neighbors()` above? since it is replace by ConnectedPeers?

Reply:
Should Neighbors() be replaced with ConnectedPeer() in `func (h *HeartbeatHandler) Log()`?

Reply:
yes, let's replace